Gemfile

```diff
@@ -9,6 +9,8 @@ gem "config"
 gem "bootstrap-sass", "3.3.7"
 gem "jquery-rails"
 gem "rails-i18n"
+gem "carrierwave", "1.2.2"     
+gem "mini_magick", "4.7.0"
```

Comment:
are you really need this gem?

Reply:
I can't speak English

Reply:
Can I speak Vietnamese?!!

Reply:
+ gem "carrierwave", "1.2.2" : để đăng ảnh ạ 
+ gem "mini_magick" :
  -thay đổi kích thước
  -định dạng
  - xoay ảnh
  -tạo bản sao của ảnh ạ